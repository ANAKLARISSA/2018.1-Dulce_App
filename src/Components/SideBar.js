import React from 'react';
import {View} from 'react-native';

class SideBar extends React.Component {

  render() {
    return (
      <View style={{flex: 0.25, backgroundColor: '#fd7d01', alignItems: 'center', justifyContent: 'center'}}>
        <View style={{backgroundColor: '#f1f1f1', alignItems: 'center', margin: 1, borderRadius: 5, width: 7.5, height: 40}} />
      </View>
    );
  }
}

export default SideBar;
