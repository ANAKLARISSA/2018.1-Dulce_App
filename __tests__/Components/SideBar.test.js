import React from 'react';
import 'react-native';
import SideBar from '../../src/Components/SideBar';
import renderer from 'react-test-renderer';

require('bezier');

it('renders correctly', () => {
  const tree = renderer
  .create(<SideBar />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
