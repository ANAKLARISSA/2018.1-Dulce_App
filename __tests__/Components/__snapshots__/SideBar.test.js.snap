// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fd7d01",
      "flex": 0.25,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#f1f1f1",
        "borderRadius": 5,
        "height": 40,
        "margin": 1,
        "width": 7.5,
      }
    }
  />
</View>
`;
